### PR TITLE
reduce memory usage of voice states

### PR DESCRIPTION
**Pull request by ghost**
(✿◡‿◡)

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@ rules:
   semi: [2, "always"]
   require-atomic-updates: 1
   no-prototype-builtins: 0
+  quotes: ["error", "double"]
```
Comment (alex-taxiera) [inline .eslintrc.yml]: use `2` instead of `"error"`

**Review by alex-taxiera — CHANGES_REQUESTED** [review]

File: lib/structures/Member.js [hunk 2]
```diff
@@ -76,6 +83,10 @@ class Member extends Base {
         }
     }
 
+    get voiceState() {
+        return this.guild.voiceStates.get(this.id) || new VoiceState({ id: this.id });
```
Comment (abalabahaha) [inline lib/structures/Member.js]: Most of the rest of the lib doesn't do one-line object definitions

File: .eslintrc.yml [hunk 3]
```diff
@@ -12,3 +12,4 @@ rules:
   semi: [2, "always"]
   require-atomic-updates: 1
   no-prototype-builtins: 0
+  quotes: [2, "double"]
```
Comment (abalabahaha) [inline .eslintrc.yml]: This should have been a separate PR (or folded into #549)

**Review by abalabahaha — CHANGES_REQUESTED** [review]

File: lib/structures/Member.js [hunk 4]
```diff
@@ -76,6 +82,11 @@ class Member extends Base {
         }
     }
 
+    get voiceState() {
+        const voiceState = this.guild.voiceStates.get(this.id);
+        return voiceState ? voiceState : new VoiceState({id: this.id});
```
Comment (Skillz4Killz) [inline lib/structures/Member.js]: How about merging these into 1 line? Would probably look a lot cleaner in my opinion.
```js
return this.guild.voiceStates.get(this.id) || new VoiceState({ id: this.id })
```

Comment (ghost) [reply lib/structures/Member.js]: https://github.com/abalabahaha/eris/pull/548#discussion_r340955586

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**